internal/indexer/definitions/ptp.yaml

```
@@ -93,31 +64,113 @@ irc:
-            torrentName: That.Movie.1972.1080p.BluRay.FLAC.x264-GROUP
-            tags: comedy, drama, romance
-        - line: That Other Movie [1972] | x264 / Blu-ray / MKV / 1080p / Freeleech! | 204371 | 964303 | That.Other.Movie.1972.1080p.BluRay.FLAC.x264-GROUP | comedy, drama, romance
+            announceType: 1
+            ptpFL: 0
+            scene: 0
+            category: 1
+            groupId: "357805"
+            torrentId: "1332009"
+            key: "9k6p"
+            imdb: "tt24249072"
+            torrentSize: "1899875175"
+            releaseTags: H.264/MKV/WEB/720p
+            editionTags: ""
+            year: "2023"
+            title: "Last Straw"
+            torrentName: Last.Straw.2023.REPACK.720p.AMZN.WEB-DL.DDP5.1.H.264-FLUX
+            tags: thriller,horror
+        - line: 1:0:0:1:357805:1332009:9k6p:tt24249072:1899875175|H.264/MKV/WEB/720p||2023|Last Straw|Last.Straw.2023.REPACK.720p.AMZN.WEB-DL.DDP5.1.H.264-FLUX|thriller,horror
           expect:
-            year: "1972"
-            releaseTags: x264 / Blu-ray / MKV / 1080p / Freeleech!
-            freeleech: Freeleech
-            torrentId: "964303"
-            torrentName: That.Other.Movie.1972.1080p.BluRay.FLAC.x264-GROUP
-            tags: comedy, drama, romance
-        pattern: '.* \[(.*)\] (?:by .*)?\| (.*?(?: \/ (Freeleech)!)?) \| .* \| (.*) \| (.*) \| (.*)'
+            announceType: 1
+            ptpFL: 0
+            scene: 0
+            category: 1
+            groupId: "357805"
+            torrentId: "1332009"
+            key: "9k6p"
+            imdb: "tt24249072"
+            torrentSize: "1899875175"
+            releaseTags: H.264/MKV/WEB/720p
+            editionTags: ""
+            year: "2023"
+            title: "Last Straw"
+            torrentName: Last.Straw.2023.REPACK.720p.AMZN.WEB-DL.DDP5.1.H.264-FLUX
+            tags: thriller,horror
+        - line: 1:0:0:1:21108:1332339:ncfe:tt0107426:63996413804|BD66/m2ts/Blu-ray/2160p|Dolby Atmos/Dolby Vision|1993|Little Buddha|Little.Buddha.1993.2160p.FRA.UHD.Blu-ray.DV.HDR.HEVC.DTS-HD.MA.5.1|drama,italian
+          expect:
+            announceType: 1
+            ptpFL: 0
+            scene: 0
+            category: 1
+            groupId: "21108"
+            torrentId: "1332339"
+            key: "ncfe"
+            imdb: "tt0107426"
+            torrentSize: "63996413804"
+            releaseTags: BD66/m2ts/Blu-ray/2160p
+            editionTags: "Dolby Atmos/Dolby Vision"
+            year: "1993"
+            title: "Little Buddha"
+            torrentName: Little.Buddha.1993.2160p.FRA.UHD.Blu-ray.DV.HDR.HEVC.DTS-HD.MA.5.1
+            tags: drama,italian
+        # <boolnew>:<enum_fl>:<boolscene>:<category>:<gid>:<tid>:<key>:tt<imdbid>:<sizebytes>|<format>/<container>/<source>/<resolution>|<edition_tags>|<year>|<title>|<releasename>|<taglist>
+        pattern: '(\d):(\d):(\d):(\d):(\d+):(\d+):(.+):(tt?\d+):(\d+)\|(.*)\|(.*)\|(\d+)\|(.+)\|(.+)\|(.+)'
```

```suggestion
        pattern: '(\d):(\d):(\d):(\d):(\d+):(\d+):(\w+):(tt\d+)?:(\d+)\|(.*)\|(.*)\|(\d+)\|(.+)\|(.+)\|(.+)'
```
Updated regex example with some change suggestions 😄 
https://regex101.com/r/8DvvNK/3